doc/en/user/source/community/cog/cog.rst

```diff
@@ -0,0 +1,134 @@
+.. _cog_plugin:
+
+COG (Cloud Optimized GeoTIFF) Support
+=====================================
+
+`COG <https://github.com/cogeotiff/cog-spec/blob/master/spec.md>`_ (Cloud Optimized GeoTIFF) is a regular GeoTIFF file, aimed at being hosted on a HTTP file server, whose internal organization is friendly for consumption by clients issuing HTTP GET range requests. 
+The COG module allows to set configuration params to connect to a Cloud GeoTIFF as well as adding JARs to the classpath needed to support that.
```

Comment:
The COG module allows setting configuration parameters to connect to a Cloud GeoTIFF, as well as adding JARs to the classpath needed to support that connection.